library/Mockery/Mock.php

```diff
@@ -20,6 +20,8 @@
 
 namespace Mockery;
 
+use Closure;
+use Mockery;
```

Comment:
I think these two lines need removing to get the tests running.

Reply:
@davedevelopment Thanks that got the tests working! So if I add some tests will this be accepted?